blog/2024-02-29-config-entry-reconfigure-step.md

```diff
@@ -0,0 +1,30 @@
+---
+author: G Johansson
+authorURL: https://github.com/gjohansson-ST
+title: "Config Entries can now provide a reconfigure step"
+---
+
+As of Home Assistant Core 2024.4 integrations can add a `reconfigure` step in their config entries to allow a user to reconfigure necessary config entry data.
```

Comment:
Some integrations already implement reconfigure by updating the configuration of an existing config entry if the user adds a matching config entry again, we should explain this is not needed. We should also clarify when reauth should be used and when reconfigure should be used.